testsuite/kuadrant/policy/dns.py

```diff
@@ -31,3 +31,10 @@ def create_instance(
         }
 
         return cls(model, context=cluster.context)
+
+    def wait_for_full_enforced(self):
```

Comment:
Instead of rewriting the `wait_for_full_enforce` method, why don't we add a parameter to the parent method for timeout:  `Policy.wait_for_full_enforce(self, timeout=60)`?

Reply:
I think timelimit should be global. It makes sense to me to have different timelimits for different policies, specifically for DNS where creation of a large amount of dnsrecord takes some time @jsmolar

Reply:
Yes, in my head the best approach is to have default timelimit that is ok for all policies in vast majority of tests and for a few tests where for certain policy longer timelimit is required (eg. dnspolicy wait till lot of dnsrecords are created or tlspolicy wait till Lets Encrypt issues a cert) the longer timelimit is explicitly set -  however, I would personally avoid overriding when implementing this if possible but that's probably just a matter of taste